scripts/publish-version.sh

```diff
@@ -67,12 +67,13 @@ git tag v${new_version}
 
 # only make remote changes if running in CI
 if [ "${CI:-}" = "true" ] && [ -z "${ACT:-}" ]; then
-  # push changes back onto origin including new tag
-  git push origin "${branch}" --tags
-
   # publish each workspace package to npm
   if [ -n "${NODE_AUTH_TOKEN:-}" ]; then
     yarn config set npmAuthToken "${NODE_AUTH_TOKEN:-}"
   fi
   yarn workspaces foreach -piv --no-private --topological-dev npm publish --access public
+
+  # we push changes back onto origin only if the npm publish step worked
```

Comment:
I don't get it, surely the versioning in github needs to happen regardless, and we should fix the npm publish?

Reply:
Good point, I changed the approached.